app/services/depositor.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# Completes the deposit of an object
+class Depositor
```

Comment:
Can we call this `DepositCompleteService` or smiliar? When I first read this, I thought it was referring to the person who was performing the deposit (like Contributor or Creator). And then I thought it was performing the deposit. Neither of which are correct.

Reply:
Yes, good call.

Reply:
@justinlittman I'm inclined to go with `DepositCompleter` if that sounds better to you.